kDrive/UI/Controller/Menu/ParameterTableViewController.swift

```diff
@@ -192,6 +192,13 @@ class ParameterTableViewController: UITableViewController {
         }
         self.driveFileManager = driveFileManager
     }
+
+    static func instantiate(driveFileManager: DriveFileManager) -> ParameterTableViewController {
```

Comment:
If we want to test this, we might want to sur the usual pattern { sevice / protocol / DI } instead of `static func`.